### ISSUE TITLE
WrongFragmentContainerViolation in Fragment StrictMode

### ISSUE THREAD

**Issue by Alireza-Farahani** [ChuckerTeam/chucker#708]
## :writing_hand: Describe the bug
I enabled Fragment StrictMode for our app ([link](https://developer.android.com/guide/fragments/debugging)), setting to `penaltyDeath` in debug mode. I got this upon clicking chucker notification
```
E/FragmentStrictMode: Policy violation with PENALTY_DEATH in com.chuckerteam.chucker.internal.ui.transaction.TransactionListFragment
    androidx.fragment.app.strictmode.WrongFragmentContainerViolation: Attempting to add fragment TransactionListFragment{6ef57b} (3d0644f8-38d5-4018-b279-79ffea651e6f id=0x7f0a0916) to container androidx.viewpager.widget.ViewPager{62c2f98 VFED..... ......I. 0,0-0,0 #7f0a0916 app:id/viewPager} which is not a FragmentContainerView
```
And since this class is internal to library, I cant exclude it via `allowViolation`.

## :bomb: Steps to reproduce

Simple app with chucker (3.5.2) set up. 
Use AndroidX fragment library 1.4.0-alpha and later and enable strict mode just like this.
```kotlin
FragmentStrictMode.setDefaultPolicy(
            FragmentStrictMode.Policy.Builder()
                .detectFragmentReuse()
                .detectFragmentTagUsage()
                .detectRetainInstanceUsage()
                .detectSetUserVisibleHint()
                .detectTargetFragmentUsage()
                .detectWrongFragmentContainer()
                .apply {
                    if (BuildConfig.BUILD_TYPE == "debug") {
                        penaltyDeath()
                    }
                }
                .build()
```

## :wrench: Expected behavior
Getting no violation from chucker.

## :camera: Screenshots
none

## :iphone: Tech info
 - Device: Pixel 2
 - OS: Android 11
 - Chucker version: 3.5.2

**Comment by vbuberen** [comment]
Thanks for reporting. Yeah, it seems that this violation is added in the latest Fragment 1.4.0 alpha version only, so this is why we haven't caught it yet. 

We will fix it shortly and I would advise you to switch to latest snapshot version till we release another update.

**Comment by vbuberen** [comment]
Just debugged this issue a little bit. 
So, in `3.5.2` we still have that `TransactionListFragment` as 3.x releases had deprecated Throwable inspection. On `develop` branch we don't have this fragment anymore.
I added strict mode policy with such config:
```
supportFragmentManager.strictModePolicy =
            FragmentStrictMode.Policy.Builder()
                .detectFragmentReuse()
                .detectFragmentTagUsage()
                .detectRetainInstanceUsage()
                .detectSetUserVisibleHint()
                .detectTargetFragmentUsage()
                .detectWrongFragmentContainer()
                .apply {
                    if (BuildConfig.BUILD_TYPE == "debug") {
                        penaltyDeath()
                    }
                }
                .build()
```
And had no issues navigating Chucker tabs, including transaction details, which has 3 fragments inside `ViewPager`.

So there is nothing to fix at the moment. I would highly recommend you to switch to our latest snapshot version to avoid this problem.

**Comment by wbervoets** [comment]
when will the new version be released? Fragment 1.4.x stable has been released in the mean time. thx!

**Comment by wbervoets** [comment]
Tried using 4.0.0-SNAPSHOT:
- I'm now able to click on the notification without crashing
- Chucker still crashes when selecting a network transaction with
 
Fatal Exception: androidx.fragment.app.strictmode.WrongFragmentContainerViolation
Attempting to add fragment TransactionOverviewFragment{ff4002b} (ab14c612-b5e4-41c1-8a05-05a1816b3c62 id=0x7f0a037c) to container androidx.viewpager.widget.ViewPager{3b95f96 VFED..... ......I. 0,0-0,0 #7f0a037c app:id/viewPager} which is not a FragmentContainerView

I'll create a new bugreport for this.
